Makefile

```diff
@@ -651,24 +651,25 @@ remotesystem:
 	# podman server spews copious unhelpful output; ignore it.
 	rc=0;\
 	if timeout -v 1 true; then \
-		SOCK_FILE=$(shell mktemp --dry-run --tmpdir podman_tmp_XXXX);\
-		export PODMAN_SOCKET=unix://$$SOCK_FILE; \
-		./bin/podman system service --timeout=0 $$PODMAN_SOCKET > $(if $(PODMAN_SERVER_LOG),$(PODMAN_SERVER_LOG),/dev/null) 2>&1 & \
+		if ./bin/podman-remote info >/dev/null 2>&1; then \
+			echo "Error: podman server is already running";\
+			exit 1;\
+		fi;\
```

Comment:
Now that I think of it I think one downside is that on a system where podman.socket in enabled this check will always trigger.
It shouldn't be and issue for CI but for local developers it might be, anyhow I guess localsystem/remotesystem shouldn't be used locally because they delete `${HOME}/.local/share/containers ${HOME}/.config/containers` and devs should use hack/bats instead.

Reply:
Using a random socket file protects against the (very very unlikely) possibility of talking to the wrong podman. So... I'm going to close. If anyone feels this is worth revisiting, please reopen and discuss.